constants/StringConstants.js

```diff
@@ -0,0 +1,33 @@
+const StringConstants = {
```

Comment:
@cgenesoniSouthWorks I think we can just delete the reference to this and remove the file altogether. Which file is requiring this?